tests/lib/e2e/e2e-spec.ts

```diff
@@ -0,0 +1,213 @@
+import supertest from 'supertest'
+import { checkConnection, cleanDataBase, createTestUser, getAuth, testUser } from './utils'
+// Remove jest args so there is no conflict with storj-bridge
+process.argv = process.argv.slice(0, 2)
+const engine = require('../../../bin/storj-bridge')
+
+import sendGridMail from '@sendgrid/mail'
+
+jest.mock('@sendgrid/mail', () => ({
+  setApiKey: jest.fn(),
+  send: jest.fn((_, __, done) => typeof done === 'function' ? done() : Promise.resolve()),
+}))
+
+const dispatchSendGridMock = jest.fn((_, __, ___, cb) => {
+  sendGridMail.send(null as any, null as any, cb)
+})
+engine.mailer.dispatchSendGrid = dispatchSendGridMock
+
+checkConnection(engine.storage)
+
+describe('Bridge E2E Tests', () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  beforeAll(() => {
+    cleanDataBase(engine.storage)
+  })
+
+  afterAll(() => {
+    cleanDataBase(engine.storage)
+  })
+
+  describe('Buckets', () => {
```

Comment:
Please, adhere to the company testing guidelines that we have in the internal [docs](https://inxt.atlassian.net/wiki/spaces/T/pages/25460784/How+to+Testing+Guideline), including but not limited to the AAA pattern, which makes debugging and understanding of the test easier

Reply:
I added extra descriptions to the tests and organized them to follow the AAA pattern
![image](https://github.com/internxt/bridge/assets/155450112/0afc0a74-2cb8-4a4d-b9b6-1e735c43dbd1)